content/articles/2018-12-textrecipes-0-0-1.Rmarkdown

```diff
@@ -0,0 +1,84 @@
+
+You can install it by running:
+
+```{r, eval = FALSE}
+install.packages("textrecipes")
+```
+
+## New steps
+
+The steps introduces here can be split into 3 types. The steps
+
+1. converts from characters to list-columns and vice versa
+1. modifies the elements in list-columns
+1. converts list-columns to numerics
+
+this allows for great flexibility in the preprocessing tasks that can be done while staying inside the [`recipes`](https://github.com/tidymodels/recipes) framework. This also prevents having one single step having a million arguments.
+
+## Workflows
+
+First we start by creating a [`recipe`](https://tidymodels.github.io/recipes/reference/recipe.html) object from the original data.
+
+```{r}
+data("okc_text")
+rec_obj <- recipe(~ ., okc_text)
+
+rec_obj
+```
+
+The workflow in `textrecipes` so far starts with [`step_tokenize()`](https://tidymodels.github.io/textrecipes/reference/step_tokenize.html) followed by a combination of type 1 and 2 steps ending with a type 3 step. [`step_tokenize()`](https://tidymodels.github.io/textrecipes/reference/step_tokenize.html) wraps the [`tokenizers`](https://github.com/ropensci/tokenizers) package for tokenization, but other tokenization functions can be utilized using the `custom_token` argument, more information concerning arguments can be found in the documentation. The shortest possible recipes are [`step_tokenize()`](https://tidymodels.github.io/textrecipes/reference/step_tokenize.html) directly followed by a type 3 step.
+
+```{r}
+### Feature hashing done on word tokens
+rec_obj %>%
+  step_tokenize(essay0) %>% # token argument defaults to "words"
+  step_texthash(essay0)
+
+### Counting chacter occurrences
+rec_obj %>%
+  step_tokenize(essay0, token = "character") %>%
+  step_tf(essay0)
+```
+
+If one wanted to calculate the word count of the top 100 most frequently used words after stemming is performed, type 2 steps are needed. Here we use [`step_stem()`](https://tidymodels.github.io/textrecipes/reference/step_stem.html) to perform stemming using the [`SnowballC`](https://CRAN.R-project.org/package=SnowballC) package and [`step_tokenfilter()`](https://tidymodels.github.io/textrecipes/reference/step_tokenfilter.html) to only keep the most 100 frequent tokens.
+
+```{r}
+rec_obj %>%
+  step_tokenize(essay0) %>%
+  step_stem(essay0) %>%
+  step_tokenfilter(essay0, max_tokens = 100) %>%
+  step_tf(essay0)
+```
+
+For more combination please consult the documentation and the [vignette](https://tidymodels.github.io/textrecipes/articles/cookbook---using-more-complex-recipes-involving-text.html) which includes recipe examples.
```

Comment:
* combination ⇨ combinations (then add a comma after it)
* Comma after vignette before which